daemon/daemon_unix_test.go

```diff
@@ -376,3 +379,75 @@ func sysInfo(t *testing.T, opts ...func(*sysinfo.SysInfo)) sysinfo.SysInfo {
 	}
 	return si
 }
+
+func TestGetBlkioWeightDevices(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "tempDevDirForBlkioThrottleDevices")
+	assert.NilError(t, err, "create temp file")
+
+	tempFile := filepath.Join(tempDir, "dev")
+
+	defer os.RemoveAll(tempDir)
+
+	// mock a file with major 0x1FD(509 in decimal) and minor 0x130(304)
+	if err = unix.Mknod(tempFile, unix.S_IFCHR, 0x11FD30); err != nil {
```

Comment:
You can use `unix.Mkdev(major, minor)` (and define `major` and `minor` as constants to not have some "magic numbers" across the code.